libs/sdk-backend-spi/src/workspace/measures/index.ts

```diff
@@ -1,6 +1,8 @@
 // (C) 2019-2020 GoodData Corporation
 import { ObjRef } from "@gooddata/sdk-model";
 import { IMeasureExpressionToken } from "../fromModel/ldm/measure";
+import { IMeasureMetadataObject } from "../fromModel/ldm/metadata";
+import { IMeasureMetadataObjectDefinition } from "../fromModel/ldm/metadata";
 
 /**
  * Service for querying additional measures data.
```

Comment:
This is no longer entirely true, now that it allows manipulation too, please expand this